config/index.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-    secret:
-        process.env.NODE_ENV === "production" ? process.env.SECRET : "secret"
+  secret:
+    process.env.NODE_ENV === 'production' ? process.env.SECRET : 'secret'
```

Comment:
This `secret` is not `secret`. It's just like using `password` as your `password`. A random alphanumeric is better.